newdle/client/src/components/creation/timeslots/CandidateSlot.js

```diff
@@ -64,13 +64,19 @@ export default function CandidateSlot({
   onDelete,
   onChangeSlotTime,
   isValidTime,
+  text,
 }) {
   const slot = (
     <Slot
       width={width}
       pos={pos}
       moreStyles={styles['candidate']}
-      tooltip={`${startTime} - ${endTime}`}
+      tooltip={
+        <div style={{textAlign: 'center'}}>
+          <span>{`${startTime} - ${endTime}`}</span>
+          <p>{text}</p>
```

Comment:
do we really want to render  an empty `<p>` here in case there is no text?

Reply:
I can ditch it if undefined.